vtysh/vtysh.c

```diff
@@ -2652,7 +2655,7 @@ DEFUN (vtysh_write_terminal,
 
 DEFUN (vtysh_show_running_config,
        vtysh_show_running_config_cmd,
-       "show running-config [<zebra|ripd|ripngd|ospfd|ospf6d|ldpd|bgpd|isisd|fabricd|pimd>]",
+       "show running-config [<zebra|ripd|ripngd|ospfd|ospf6d|ldpd|bgpd|isisd|fabricd|pimd|staticd>]",
```

Comment:
`eigrpd`, `bfdd`, `pbrd` and `sharpd` are still missing. I think we should consider using an auto-generated string (similar to the strings generated by the _route_types.pl_ script) to avoid the need to update all these commands every time a new daemon is introduced.